py4DSTEM/braggvectors/diskdetection_aiml.py

```diff
@@ -256,7 +253,7 @@ def find_Bragg_disks_aiml_selected(
 
     try:
         import crystal4D
-    except:
+    except ModuleNotFoundError:
         raise ImportError("Import Error: Please install crystal4D before proceeding")
```

Comment:
In some places you are catching `ModuleNotFoundError` but raising `ImportError`. Why the substitution?

Reply:
It was Joydeep's error message, I'm assuming it was a choice to be more readable or it was a mistake. I was more concerned about the bare exception. I'll change it to be ModuleNotFoundError